src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+    if (boolean) {
```

Comment:
It can be refactored by using the ternary operator. 
`````
const booleanToWord = (boolean) => (boolean ? "Yes" : "No")
`````
The conditional (ternary) operator takes three operands: a condition followed by a question mark (?), then an expression to execute if the condition is truthy followed by a colon (:), and finally the expression to execute if the condition is falsy. This operator is frequently used as an alternative to an if...else  statement.